tensorlayer/layers/convolution.py

```diff
@@ -1615,12 +1615,19 @@ class DepthwiseConv2d(Layer):
 
     Examples
     ---------
-    >>> x = tf.placeholder(tf.float32, shape=[None, 28, 28, 1], name='x')
-    >>> net = InputLayer(x, name='in')
-    >>> net = Conv2d(net, 32, (3, 3), (1, 1), name='conv1')
-    >>> net = MaxPool2d(net, (2, 2), name='pool1')
-    >>> net = DepthwiseConv2d(net, (3, 3), (1, 1), act=tf.nn.relu, name='dethwise1')
-    >>> net = Conv2d(net, 64, (1, 1), (1, 1), act=tf.nn.relu, name='conv2')
+    >>> net = InputLayer(x, name='input')
+    >>> net = Conv2d(net, 32, (3, 3), (2, 2), b_init=None, name='cin')
```

Comment:
shall we use keyword argument in the sample code here?

Reply:
i think this one look simpler.